projects/okernel/Dockerfile.okernel

```diff
@@ -1,13 +1,33 @@
-FROM linuxkit/alpine-build-kernel:cfdd576c36a52ed2dd62f237f79eeedc2dd3697b@sha256:3fe08db373a9373ba1616a485858f01ebd2d7a3cb364a099d0ed8b45fa419da2
+FROM linuxkit/alpine:a120ad6aead3fe583eaa20e9b75a05ac1b3487da AS kernel-build
+RUN apk --no-cache add \
+    argp-standalone \
+    automake \
+    bash \
+    bc \
+    binutils-dev \
+    bison \
+    build-base \
+    curl \
+    diffutils \
+    git \
+    installkernel \
+    kmod \
+    libelf-dev \
+    linux-headers \
+    sed \
+    tar \
+    xz \
+    zlib-dev \
+    openssl-dev
+
+RUN apk --no-cache add --repository http://dl-cdn.alpinelinux.org/alpine/edge/main openssl
```

Comment:
This makes the build non-reproducible as it pulls an unknown version over the internet, but we can leave this for now

Reply:
Agreed, I am not happy with this either.

Somehow openssl is not found if attempting to pull it alongside the rest of the packages in the first RUN statement. No such issue with openssl-dev. Which repos is linuxkit/alpine using internally - aren't they those for v3.6?

The other alternative is to unset CONFIG_MODULE_SIG for now in the kernel configuration and remove the dependency.

Reply:
I'm adding it to `linuxkit/alpine` now, but might take a day or so before it get's merged

Reply:
Great, I'll fix the Dockerfile once the merge goes through. Thanks